support/agent/blockstore_util.go

```diff
@@ -19,7 +21,7 @@ func BlockstoreCopy(from, to cbor.IpldBlockstore, root cid.Cid) (blocks uint64,
 	cp := func(blk block.Block) error {
 		numBlocks++
 		totalCopySize += uint64(len(blk.RawData()))
-		return to.Put(blk)
+		return to.Put(context.TODO(), blk)
```

Comment:
Why?

Reply:
Because go-state-types pulls in the context blockstores (and I don't want to undo that, it should be on the latest thing).